lib/utils/config/src/test/env-reader-test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const { envReader } = require('./../lib/env-reader'); 
+
+describe('env reader tests', () => {
+  context('when env variable is exported', () => {
+    it('should return the env variable value', () => {
+      const expectedValue = 'test-var-value';
+      process.env.TEST_VAR = expectedValue;
+
+      const value = envReader('TEST_VAR');
+      delete process.env.TEST_VAR;
```

Comment:
Isn't it better this line to go in either the tests that actually depends on it, or in a "beforeEach" function